R/get_available_datasets.R

```diff
@@ -26,6 +25,33 @@
 #' # see only regional level datasets
 #' get_available_datasets("regional")
 get_available_datasets <- function(type) {
```

Comment:
I think this is a nice idea. However, do you think we could keep it all in one function and have a `render` argument so that the user just sees `get_available_datasets` and can either use the sourced data or get it from the supported classes. Even cooler (and thinking long term) if the namespace (i.e `covidregionaldata`) was a passed parameter and used to both find classes and for loading the cached version that would be very neat.

Reply:
I have removed `render_regional_data` and put the code into `get_regional_data` with a render argument (also updated tests). I have added a parameter for namespace defaulting to `covidregionaldata` but I am not sure how to set this up with caching.